Makefile

```diff
@@ -360,6 +372,12 @@ $(OBJDIR)/mfem-% : examples/mfem/%.cpp $(libceed) | $$(@D)/.DIR
 	  MFEM_DIR="$(abspath $(MFEM_DIR))" $*
 	mv examples/mfem/$* $@
 
+$(OBJDIR)/nek-% : examples/nek/%.usr examples/nek/nek-examples.sh $(libceed) | $$(@D)/.DIR
```

Comment:
If any two Nek examples conflict with each other, should this just say `nek-bps` instead of the wildcard `nek-%` (to avoid a future developer accidentally recreating the present problem)?

Reply:
Updated